regression/syscall_test/install_bazel.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
```

Comment:
It is probably better to move `install_bazel.sh` script to `tools` because (1) that folder is where utility scripts can be found and (2) installing Bazel is not directly related to running system calls, it is more about an environment setup.

If `install_bazel.sh` is under `tools/`, you also need to update the Makefile that mentions the `install_bazel` script.

Reply:
Moved the script to `/tools`